components/scream/src/physics/shoc/shoc_functions_f90.cpp

```diff
@@ -1799,7 +1826,7 @@ void check_length_scale_shoc_length_f(Int nlev, Int shcol, Real* host_dx, Real*
   // Sync to device
   std::vector<view_1d> temp_1d_d(2);
   std::vector<view_2d> temp_2d_d(1);
-  ekat::host_to_device({host_dx,host_dy}, shcol, temp_1d_d);
+  copy_to_device({host_dx,host_dy}, shcol, temp_1d_d);
   ekat::host_to_device({shoc_mix}, shcol, nlev, temp_2d_d, true);
```

Comment:
Any reason not to use `copy_to_device` here too?

Reply:
`copy_to_device` is a function I wrote for 1d scalar views. `shoc_mix` is a packed view, so it still requires the ekat function.